samples/bluetooth/channel_sounding_ras/reflector/README.rst

```diff
@@ -0,0 +1,79 @@
+.. _channel_sounding_ras_reflector:
+
+Bluetooth: Channel Sounding Reflector with Ranging Responder
```

Comment:
Is this name from a BT spec or something? If not, no need to capitalize other than the first word.

Reply:
Channel Sounding, Ranging Responder and Reflector are names from BT specs where they're capitalized as such, but I can change this if you think it's better.

Reply:
Ok, let's stick to the spec then.